src/main/java/io/anserini/embeddings/search/SearchW2V_Args.java

```diff
@@ -0,0 +1,26 @@
+package io.anserini.embeddings.search;
```

Comment:
Can you please move this as an inner class, like https://github.com/twitter/GraphJet/blob/master/graphjet-demo/src/main/java/com/twitter/graphjet/demo/PageRankCassovaryDemo.java